meta-resin-common/recipes-connectivity/usb-modeswitch/libpipeline_1.4.1.bb

```diff
@@ -0,0 +1,14 @@
+# Required by "dispatcher-c-rewrite.patch".
```

Comment:
agherzan 21 hours ago
I would avoid this comment here as it is obvious that being a dependency of USB modeswitch it is needed by it.